zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/CamundaExporterITInvocationProvider.java

```diff
@@ -42,58 +32,27 @@ public class CamundaExporterITInvocationProvider
         AfterEachCallback {
 
   public static final String CONFIG_PREFIX = "custom-prefix";
-  protected SearchClientAdapter elsClientAdapter;
-  protected SearchClientAdapter osClientAdapter;
-  private final ElasticsearchContainer elsContainer =
-      TestSearchContainers.createDefeaultElasticsearchContainer();
-  private final OpensearchContainer<?> osContainer =
-      TestSearchContainers.createDefaultOpensearchContainer();
-  private ElasticsearchClient elsClient;
-  private OpenSearchClient osClient;
-  private final List<AutoCloseable> closeables = new ArrayList<>();
-
-  protected ExporterConfiguration getConfigWithConnectionDetails(
-      final ConnectionTypes connectionType) {
-    final var config = new ExporterConfiguration();
-    config.getIndex().setPrefix(CONFIG_PREFIX);
-    config.getBulk().setSize(1); // force flushing on the first record
-    if (connectionType == ELASTICSEARCH) {
-      config.getConnect().setUrl(elsContainer.getHttpHostAddress());
-
-    } else if (connectionType == OPENSEARCH) {
-      config.getConnect().setUrl(osContainer.getHttpHostAddress());
+  private final SearchDatabaseCallbackDelegate callbackDelegate;
+
+  public CamundaExporterITInvocationProvider() {
+    final var shouldRunAgainstAWSOS =
+        Optional.ofNullable(System.getProperty(IT_OPENSEARCH_AWS_INSTANCE_URL_PROPERTY))
```

Comment:
:wrench: Let's document this in the class Javadoc so it's clear how this invocation provider is meant to be used, how it can be configured, etc.